authz/rbac_translator.go

```diff
@@ -304,12 +304,9 @@ func (options *auditLoggingOptions) toProtos() (allow *v3rbacpb.RBAC_AuditLoggin
 
 	for i := range options.AuditLoggers {
```

Comment:
`for _, config := range options.AuditLoggers`?

Reply:
Changed, I think this was a holdover from when this was structured differently.

Reply:
Looks like this is why?

> `range var config copies lock: google.golang.org/grpc/authz.auditLogger contains google.golang.org/protobuf/types/known/structpb.Struct contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex`

Should this be `[]*auditLogger` then?

Reply:
Oh yes, I remember now that is why I had done that - changed to `[]*auditLogger`